src/Http/Controllers/ContentfulSyncController.php

```diff
@@ -92,7 +93,7 @@ public function handleIncomingWebhook(Request $request): Response
             case self::TOPIC_CONTENT_MANAGEMENT_ENTRY_DELETE:
                 /** @var Entry $resource */
```

Comment:
This should probably be DeletedEntry too?

Reply:
Good catch! Updated it as well

Reply:
@Jalle19 Hello! Had a time to review?